### DISCUSSION TITLE
Signal 11 error during RC extraction

### DISCUSSION THREAD

**Discussion by faisaladilquadery** [The-OpenROAD-Project/OpenROAD-flow-scripts#1583] [Q&A]
I am facing a error for a opensource design or1200 immu top and pdk TSMC65 in the rc extraction stage. The design frequency is 2GHz and the netlist was synthesized from Cadence genus and used in ORFS to implement.
Can anyone deduce the cause please ?
![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/140678965/5b07a323-7c9a-48db-87ba-b4aaef34d36c)

**Comment by maliberty** [comment]
tmg_conn is one of the worst places to get a crash as that code is quite obscure.  It is very hard to know without having a test case.

You might try to reduce the problem using https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/etc/deltaDebug.py to see if it becomes more obvious in a smaller netlist.
